pbs/pbsrequest.go

```diff
@@ -158,6 +158,7 @@ func ParsePBSRequest(r *http.Request, cache cache.Cache) (*PBSRequest, error) {
 		if anid, err := r.Cookie("uuid2"); err == nil {
 			pbsReq.Cookie.TrySync("adnxs", anid.Value)
 		}
+		pbsReq.User.ID = pbsReq.GetUserID("adnxs")
```

Comment:
Just double-checking my understanding...

is this only in the `pbsReq.App == nil` block because `getUserID` will _always_ return an empty string for Apps?

Reply:
Yup `getUserID` will always return an empty string for mobile apps because we don't add any cookies on the request from Prebid Mobile to Prebid Server. Additionally, Prebid Server does not attempt to parse cookies from requests that have a non nil App.